GettingStarted.md

```diff
@@ -48,24 +48,24 @@ Include the Oboe header:
 
     #include <oboe/Oboe.h>
 
-Streams are built using an `OboeStreamBuilder`. Create one like this:
+Streams are built using an `StreamBuilder`. Create one like this:
```

Comment:
an -> a

Reply:
Deferring doing the an/a changes until we have made a decision on `Stream` vs `AudioStream` naming